### ISSUE TITLE
No roles_and_accessibility identifiers

### ISSUE THREAD

**Issue by acbegen** [Dash-Industry-Forum/Dash-Industry-Forum.github.io#26]
https://dashif.org/identifiers/introduction/ links to a roles and accessibility identifiers page but there is no such page. If that is the case, the link should be removed.
